packages/core/bootstrap/src/lib/ws/index.ts

```diff
@@ -18,9 +18,11 @@ export const withWebSockets = (
 
   const wsHandler = await makeWsHandler()
   const wsConfig = getWSConfig()
+  console.log(wsConfig, wsHandler)
```

Comment:
This seems as debug log messages, and this should already be logged by WS epics.
Consider removing this extra logging.

Reply:
It _is_ extra logging that shouldn't have been committed, removing.